core/widgetdiv.js

```diff
@@ -147,6 +146,5 @@ Blockly.WidgetDiv.position = function(anchorX, anchorY, windowSize,
   }
   Blockly.WidgetDiv.DIV.style.left = anchorX + 'px';
   Blockly.WidgetDiv.DIV.style.top = anchorY + 'px';
-  Blockly.WidgetDiv.DIV.style.height =
-      (windowSize.height - anchorY + scrollOffset.y) + 'px';
+  Blockly.WidgetDiv.DIV.style.height = windowSize.height + 'px';
```

Comment:
Reverting back to the old widget height will cause issues with multiple or scrolled workspaces.